js/base/Exchange.js

```diff
@@ -2056,6 +2056,30 @@ module.exports = class Exchange {
         return rate;
     }
 
+    handleOptionAndParams (methodName, params, optionName, defaultValue = undefined) {
```

Comment:
@ttodua don't you think the parameter's order is a bit weird?

usually we put `params` as the first or last parameter

```
this.safeX(params, Y)
createOrder(..., params)
```
In this case, the last slot is taken by the optional argument, so I would suggest either placing it as the first one or removing the `defaultValue` and keeping it as the last

thoughts?

Reply:
you are right, good catch. Actually, removing `defaultValue` doesn't seem good to me, so the less intrusive approach would be placing `params` as the first argument, and checking it against undefined just for safety.

Reply:
I've pushed fix, please re-review.